data/api/client-server/relations.yaml

```
@@ -294,6 +297,27 @@ components:
       example: org.example.my_relation
       schema:
         type: string
+    recurse:
+      in: query
+      name: recurse
+      x-addedInMatrixVersion: "1.10"
+      required: false
+      description: |-
+        Whether to additionally include events which only relate indirectly to the
+        given event, ie. events related to the root events via one or more direct relationships.
```

https://github.com/matrix-org/matrix-spec/blob/main/meta/documentation_style.rst#general

```suggestion
        given event, i.e. events related to the root events via one or more direct relationships.
```

---

also: *two* or more relationships, no?

---

also also: we don't seem to use the term "root event" elsewhere: should this be "parent event" ?

---

Done in 1ba0ab9d. It's worth noting that the original was the text that the MSC proposed to add, so hopefully it's clear here that this is actually what it meant, but ideally we should have caught this in MSC review.